### DISCUSSION TITLE
Handling Exceptions while streaming

### DISCUSSION THREAD

**Discussion by m11ad** [tweepy/tweepy#1576] [Questions] [answered]
Hi, I get the exception bellow in the line `    stream.filter(track=keywords, languages=["en"])`

**Exception has occurred: ProtocolError
('Connection broken: IncompleteRead(4 bytes read)', IncompleteRead(4 bytes read))**

```python
def main(keywords):
    api = create_api()
    tweets_listener = FavRetweetListener(api)
    stream = tweepy.Stream(api.auth, tweets_listener)
    stream.filter(track=keywords, languages=["en"])


```

How should I handle this?

**Comment by Harmon758** [comment, marked as answer]
This should already be fixed in the development version on the master branch with https://github.com/tweepy/tweepy/commit/68e19cc6b9b23d72369ca1520093770eb18a5a9f.
See also the relevant issues closed with that commit: #237, #448.

For code block usage, see https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks.
